File: java/cucumber-archetype/src/main/java/postoffice/models/Label.java
package postoffice.models;

public class Label {
    private UUID id;
    private Package pkg;
    private double totalCost;

    // Constructor
    public Label(UUID id, Package pkg, double totalCost) {
        this.id = id;
        this.pkg = pkg;
        this.totalCost = totalCost
    }

    // Getters
    public UUID getId() {
        return id;
    }

    public Package getPackage() {
        return pkg;
    }

    public double getTotalCost(){
        return totalCost;
    }

    // Setters
    public void setId(UUID id) {
        this.id = id;
    }

    public void setPackage(Package pkg) {
        this.pkg = pkg;
    }

    public void setTotalCost(double totalCost){
        this.totalCost = totalCost;
    }

    @Override
    public String toString() {
        return "Label{" +
                "id=" + id +
                "totalCost=" + totalCost +
                ", packageId=" + (pkg != null ? pkg.getId() : "null") +
                '}';
    }
}

File: node/PostOffice/src/models/Label.ts
import {Package} from "./Package";

export class Label{
  private id: string;
  private package: Package;
  private totalCost: number;

  constructor(id: string, pkg: Package, totalCost: number) {
    this.id = id;
    this.package = pkg;
    this.totalCost = totalCost;
  }

  // Getters
  public getId(): string {
    return this.id;
  }

  public getPackage(): Package {
    return this.package;
  }

  public getTotalCost(): number {
    return this.totalCost;
  }

  // Setters
  public setId(id: string): void {
    this.id = id;
  }

  public setPackage(pkg: Package): void {
    this.package = pkg;
  }

  public setTotalCost(totalCost: number): void {
    this.totalCost = totalCost;
  }
}